components/agenda-jeune/OngletAgendaBeneficiaire.tsx

```diff
@@ -0,0 +1,104 @@
+import { DateTime } from 'luxon'
+import React, { useEffect, useState } from 'react'
+
+import AgendaActionRow from './AgendaActionRow'
+
+import { AgendaRdvRow } from 'components/agenda-jeune/AgendaRdvRow'
+import { IntegrationPoleEmploi } from 'components/jeune/IntegrationPoleEmploi'
+import { SpinningLoader } from 'components/ui/SpinningLoader'
+import { Action } from 'interfaces/action'
+import { RdvListItem } from 'interfaces/rdv'
+import { fusionneEtTriActionsEtRendezVous } from 'presentation/AgendaJeunePresentationHelper'
+import { AgendaMetadata, AgendaService } from 'services/agenda.service'
+import { toShortDate } from 'utils/date'
+import { useDependance } from 'utils/injectionDependances'
+
+interface OngletAgendaBeneficiaireProps {
+  idJeune: string
```

Comment:
```suggestion
  idBeneficiaire: string
```
si on reste bien sur ce terme, en cours de migration partout